### ISSUE TITLE
The user creation API is not documented

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#10868]
### Problem
- The user creation API is not documented

**Comment by hangy** [comment]
I think it's not supposed to be documented as it's only internal for the Flutter app? IMHO it should be deprecated after most users of the Flutter app have moved to Keycloak, so that it can be removed. After moving to Keycloak, user registration should not be the responsibility of the OFF API any more, that I'd probably skip documenting a functionality that's supposed to be deprecated soon.

**Comment by teolemon** [comment]
Agreed, but at least a page in the main doc explaining that would be great. 
My starting point was the NodeJS SDK. 
Also, should https://openfoodfacts-explorer.vercel.app/settings or projects like that be able to empower user creation ?

**Comment by hangy** [comment]
> Also, should https://openfoodfacts-explorer.vercel.app/settings or projects like that be able to empower user creation ?

Yes ideally by redirecting to Keycloak, showing it as an `iframe` or similar. Especially with the idea of requiring email validation and allowing users to use social login providers or Passkeys to register or sign in, the best practice would be to enforce the use of the Keycloak UI for that as much as possible.
